rider/Digma.Rider.Plugin/Digma.Rider/Util/ReflectionUtils.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using Digma.Rider.Logging;
+using JetBrains.ProjectModel;
+using JetBrains.ProjectModel.Model2.Assemblies.Impl;
+using JetBrains.Rider.Model;
+using JetBrains.Util;
+using Assembly = System.Reflection.Assembly;
+
+namespace Digma.Rider.Util;
+
+public class ReflectionUtils
```

Comment:
looks like a dummy class...

Reply:
yes its not necessary just an example for me, i will delete